docs/_plugins/resolve_permalink.rb

```diff
@@ -0,0 +1,29 @@
+module ResolvePermalink
+  # Resolve a permalink URI into a page or collection document:
+  #
+  # {{ '/' | resolve_permalink %} => home page
+  # {{ '/components/foo/' | resolve_permalink %} => from a collection
+  def resolve_permalink(href)
```

Comment:
This implementation is not new, it already existed in the now-removed `uswds.rb`. This is the only utility from that module we still needed, and the naming "USWDS" was confusing, so I renamed the file and module to reflect the single remaining helper.